modelci/hub/converter/to_trt.py

```diff
@@ -32,11 +34,42 @@
 )
 from modelci.utils import Logger
 
+
 logger = Logger('converter', welcome=False)
 
 
 class TRTConverter(object):
-    supported_framework = ["onnx", "tfs"]
+    supported_framework = ["onnx", "tfs", "tensorflow"]
+
+
+    @staticmethod
+    def from_tensorflow(
+            savedmodel_path: Path,
+            shape: List,
+            opset: int = 10,
+    ):
+        """
+        This is the function to create the TensorRT engine
+        Args:
+           savedmodel_path : Path to savedmodel_file.
+           shape : Shape of the input of the savedmodel file.
+       """
+        import tensorrt as trt
+
+        tmpdir = tempfile.mkdtemp()
+
+        onnx_save = str(os.path.join(tmpdir, "temp_from_tf/")) + '/tempmodel.onnx'
+        convertcmd = ['python', '-m', 'tf2onnx.convert', '--saved-model', savedmodel_path, '--output', onnx_save,
+                      '--opset', str(opset)]
+        subprocess.run(convertcmd)
+        TRT_LOGGER = trt.Logger(trt.Logger.WARNING)
+        with trt.Builder(TRT_LOGGER) as builder, builder.create_network(1) as network, trt.OnnxParser(network, TRT_LOGGER) as parser:
```

Comment:
So this is still ONNX -> TRT?

Reply:
Yes. Essentially it combines tf2onnx and onnx2trt. I have tested this process on lenet5, mobilenetv2 and resnet50 and got good results which proved feasibility.

Reply:
Have you test LSTM / attention / transformer? Also, I believe it is better only to have a `from_onnx` function, and leave TF -> TRT conversion solved by some graph search algorithm. What do you think @univerone

Reply:
Should I withdraw this tf2trt function?